bridge/client/app/_components/ktb-create-secret-form/ktb-create-secret-form.component.html

```diff
@@ -0,0 +1,50 @@
+<div fxFlexFill>
+  <div class="container" fxFlex fxLayout="column">
+    <div class="mb-3 settings-section" fxFlex fxLayout="column">
+      <h2>Create Secret</h2>
+      <p class="mt-0 mb-3">
+        Create a secret to store e.g. credentials for the Keptn API or other integrations.
+      </p>
+
+      <form [formGroup]="createSecretForm">
+        <div class="mb-3" fxLayout="row" fxLayoutAlign="none end" fxLayoutGap="15px">
+          <dt-form-field>
+            <dt-label>Name</dt-label>
+            <input [(ngModel)]="secret.name" formControlName="name" type="text" required dtInput placeholder="Secret name" />
```

Comment:
Secrets have to be lower case.
`a DNS-1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')`
The loading icon for creating the secret is still there after an error